lib/oktakit/client.rb

```diff
@@ -182,7 +187,8 @@ def sawyer_agent
         http.headers[:accept] = 'application/json'
         http.headers[:content_type] = 'application/json'
         http.headers[:user_agent] = "Oktakit v#{Oktakit::VERSION}"
-        http.authorization 'SSWS ', @token
+        http.authorization 'SSWS ', @token if @token
+        http.authorization :Bearer, @access_token if @access_token
```

Comment:
Should this be an either or?

Reply:
Yes. What do you think should be the higher priority from the API key and access token, in case both are passed in. Or should we raise an error in that case?

Reply:
Good question! I'd say raise an error in the case when both are passed in, much like we raise in error if none of them are passed in.

Reply:
Done. On line 39. Does it look fine now?

Reply:
Pinging for an update.